torch/_inductor/debug.py

```diff
@@ -451,6 +451,7 @@ def ir_post_fusion(self, nodes: SchedulerNodeList):
 
     def _write_ir(self, filename: str, nodes: SchedulerNodeList):
         with self.fopen(filename) as fd:
+            print(f"Writing debug ir to {fd.name}")
```

Comment:
Not needed I guess?
```suggestion
```

Reply:
This is only generated if debug logs are already set. Currently we were writing them out but not specifying where they were written so you couldnt find them. I can switch this to using logging though.

Reply:
Still I think `print` is worse than using a structured logging (@ezyang knows best which mechanism is preferred right now)